### PR TITLE
1letter/fix #1632

### LINKED ISSUE #1632: Update requirements to latest Plone 6.0.4

### PR DESCRIPTION
**Pull request by 1letter**
Update requirements.txt and plone6 buildout to Plone 6.0.4

### CODE REVIEW

**Comment by tisto** [comment]
@1letter you need to update the docs, this is why the Plone 6.0, Python 3.9 job fails (this jobs tests that the docs are in sync).

**Comment by 1letter** [comment]
@jenkins-plone-org please run jobs

File: news/1632.misc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Update buildout and requirements to Plone-6.0.4
+[1letter]
```
Comment (stevepiercy) [inline news/1632.misc:2]:
```suggestion
```

File: news/1632.misc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Update buildout and requirements to Plone-6.0.4
```
Comment (stevepiercy) [inline news/1632.misc:1]: We use `@github_username` format now.

```suggestion
Update buildout and requirements to Plone-6.0.4. @1letter
```

Comment (stevepiercy) [reply news/1632.misc:1]: Also `.misc` is not a valid file extension. See https://github.com/plone/plone.restapi/blob/master/pyproject.toml for valid extensions. This should be `1632.bugfix`.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: news/1632.misc (hunk 2 above)
Comment (tisto) [reply news/1632.misc:1]: @1letter @stevepiercy I'd suggest using the "internal" file extension here.

File: src/plone/restapi/tests/http-examples/controlpanels_get_rule.resp [hunk 3]
```diff
@@ -350,7 +350,6 @@ Content-Type: application/json
                 "required": [
                     "subject",
                     "recipients",
-                    "exclude_actor",
```
Comment (tisto) [inline src/plone/restapi/tests/http-examples/controlpanels_get_rule.resp:353]: Just out of curiosity. Does anyone know why "exclude_actor" was removed in Plone 6.0.4? cc @mauritsvanrees

File: news/1632.misc (hunk 2 above)
Comment (stevepiercy) [reply news/1632.misc]: You are correct. `.internal` is more appropriate.

File: src/plone/restapi/tests/http-examples/controlpanels_get_rule.resp (hunk 3 above)
Comment (mauritsvanrees) [reply src/plone/restapi/tests/http-examples/controlpanels_get_rule.resp:353]: This is for content rules, right?  No idea, I never use them. 
This is the diff in versions:

```
6.0.3.cfg:plone.app.contentrules = 5.0.0
6.0.4.cfg:plone.app.contentrules = 5.0.2
6.0.3.cfg:plone.contentrules = 2.1.3
6.0.4.cfg:plone.contentrules = 3.0.0
```

Seems to be just updating config files, dropping Py 2 support, stuff like that.

**Review by tisto — APPROVED** [review]
@1letter LGTM. I'd just rename .misc to .internal.

**Comment by 1letter** [comment]
@jenkins-plone-org please run jobs

**Review by stevepiercy — APPROVED** [review]
News is fine now, but I have no idea about the rest of the PR. Someone else should review.

**Pull request merged**